### PR TITLE
Open top-level type documentation for methods.

### LINKED ISSUE #53: cdk/browse on static methods doesn't work
When I try this:

``` clojure
(cdk/browse Alarm/addAlarmAction)
```

The function opens the root of my project on my local machine.

cdk-clj SHA is ec5a52245a725804e6a4a5db1cd4ae0493a0e679.

### PR DESCRIPTION
**Pull request by enocom**
Fixes #53.

### CODE REVIEW

**Comment by enocom** [comment]
@tvanhens I could do some more work to open a URL with the correct fragment, but have stopped here to make sure the approach makes sense.

**Review by aesterline — APPROVED** [review]
LGTM

File: src/stedi/cdk/import.clj [hunk 1]
```diff
@@ -26,35 +26,43 @@
...
+                                      {:class       fqn
+                                       :method-name name
+                                       :args        args
+                                       :intern-args intern-args} e)))))
+                fn-meta))
+      (intern ns-sym
+              (with-meta (symbol name)
+                {:doc      (render-docs docs)
+                 :arglists (list (vec (cons 'this (mapv (comp symbol :name) parameters))))})
+              (with-meta
```
Comment (tvanhens) [inline src/stedi/cdk/import.clj]: The metadata should go on the symbol above (which will get put on the var that gets bound) rather than the function.

Comment (enocom) [reply src/stedi/cdk/import.clj]: I'll make the change.

**Comment by enocom** [comment]
Some upcoming work from @tvanhens will conflict with this change. I'm going to re-open this once that other work lands.

**Pull request closed without merging**